stripper/ze_cyberspace_v1.cfg

```diff
@@ -1,3 +1,17 @@
+;lazy way to patch stage 3 nuke avoidance spot(s)
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"targetname" "c3_end_lv_change"
+	}
+	insert:
+	{
+		"OnTrigger" "playerRunScriptCodeforeach(k,_ in{SetHealth=0}){if(self.GetTeam()==2&&self.GetHealth()>0)EntFireByHandle(self,k,(0).tostring(),0,null,null)}101"
+	}
+}
```

Comment:
Do we know that humans can't reach this nuke avoidance spot? This won't fix anything if a human is the one abusing it.